src/staticanalysis/bot/tests/mocks/config.yaml

```diff
@@ -7,6 +7,7 @@ clang_checkers:
    publish: !!bool yes
  - name: clang-analyzer-security.*
    publish: !!bool no
+   reason: "Mandated by Mozilla's style guide: https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style#Control_Structures"
```

Comment:
Nit: Even though it would work for a test file, I find it a bit strange to say that "all security checkers are mandated by Mozilla's style guide".

Could you please move the reason field below, e.g. to "modernize-use-nullptr", and change its text to something like "Modernize our code base to C++11"? That would make more sense to me.